examples/connext_dds/printing_qos/README.md

```diff
@@ -0,0 +1,128 @@
+# Example Code: Print QoS
+
+Example about printing the QoS of the entities.
+
+## Concept
+
+RTI Connext DDS provides you the possibility to print the following Qos objects:
+`DataReaderQos`, `DataWriterQos`, `PublisherQos`, `SubscriberQos`, `TopicQos`,
+`DomainParticipantQos` and `DomainParticipantFactoryQos`.
```

Comment:
Add a sentence:

In addition to new APIs, when logging verbosity is set to atleast LOCAL, and category is set to API, the QoS configuration being used by an entity is logged when the entity is created and when the QoS is updated (via the set_qos operations).